lib/index.js

```diff
@@ -164,5 +180,6 @@ module.exports = {
   start,
   stop,
   reset,
-  mode
+  getMode,
```

Comment:
IMO, all functions should be easily differentiated from property names. This can be achieved by naming the function with a verb.